sensor_stack/sparton/src/sparton.py

```diff
@@ -6,15 +6,20 @@
 import serial
 import math
 import struct
-import numpy
+import numpy as np
 import rospy
 import sys
 import os
 from kraken_msgs.msg import imuData
+from kraken_msgs.msg import imuData_new
+from kraken_msgs.msg import magnetoTemp
+from geometry_msgs.msg import Vector3 , Quaternion
+
 
 from resources import topicHeader
 
-pub = rospy.Publisher(topicHeader.SENSOR_IMU, imuData, queue_size = 2)
+pub1 = rospy.Publisher(topicHeader.SENSOR_IMU, imuData, queue_size = 2)
+pub2 = rospy.Publisher(topicHeader.SENSOR_IMU+"_new", imuData_new, queue_size = 2)
```

Comment:
@athityakumar Define this new topic inside the `topicHeader.cpp` file and then use that here. The name can be `SENSOR_IMU_NEW`.